arctic/store/version_store.py

```diff
@@ -288,6 +287,14 @@ def _write_handler(self, version, symbol, data, **kwargs):
             handler = self._bson_handler
         return handler
 
+    def _update_handler(self, version, symbol, data, **kwargs):
+        handler = None
+        for h in _TYPE_HANDLERS:
```

Comment:
I guess for the case of updates, we already have a version document so can we just dispatch directly to the correct handler for the chunks?